components/cellery-component-test/src/test/java/org/cellery/components/test/HRTest.java

```diff
@@ -0,0 +1,113 @@
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.cellery.components.test;
+
+import io.cellery.models.Cell;
+import org.ballerinax.kubernetes.exceptions.KubernetesPluginException;
+import org.ballerinax.kubernetes.utils.KubernetesUtils;
+import org.cellery.components.test.utils.CelleryUtils;
+import org.cellery.components.test.utils.LangTestUtils;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static org.cellery.components.test.utils.CelleryTestConstants.CELLERY;
+import static org.cellery.components.test.utils.CelleryTestConstants.CELLERY_IMAGE_NAME;
+import static org.cellery.components.test.utils.CelleryTestConstants.CELLERY_IMAGE_ORG;
+import static org.cellery.components.test.utils.CelleryTestConstants.CELLERY_IMAGE_VERSION;
+import static org.cellery.components.test.utils.CelleryTestConstants.CELLERY_MESH;
+import static org.cellery.components.test.utils.CelleryTestConstants.EMPLOYEE_PORTAL;
+import static org.cellery.components.test.utils.CelleryTestConstants.TARGET;
+
+public class HRTest {
+    private static final Path SAMPLE_DIR = Paths.get(System.getProperty("sample.dir"));
+    private static final Path SOURCE_DIR_PATH = SAMPLE_DIR.resolve(EMPLOYEE_PORTAL + "/" + CELLERY + "/" + "hr");
+    private static final Path TARGET_PATH = SOURCE_DIR_PATH.resolve(TARGET);
+    private static final Path CELLERY_PATH = TARGET_PATH.resolve(CELLERY);
+    private Cell cell;
+
+    @BeforeClass
+    public void compileSample() throws IOException, InterruptedException {
+        String imgData = "{\"org\":\"test-org\", \"name\":\"test-img12\", \"ver\":\"1.3.5\"}";
+        Assert.assertEquals(LangTestUtils.compileBallerinaFile(SOURCE_DIR_PATH, "hr.bal", imgData), 0);
+        File artifactYaml = CELLERY_PATH.resolve("test-img12.yaml").toFile();
+        Assert.assertTrue(artifactYaml.exists());
+        cell = CelleryUtils.getInstance(CELLERY_PATH.resolve("test-img12.yaml").toString());
+    }
+
+    @Test
+    public void validateCellAvailability() {
+        Assert.assertNotNull(cell);
+    }
+
+    @Test
+    public void validateAPIVersion() {
+        Assert.assertEquals(cell.getApiVersion(), "mesh.cellery.io/v1alpha1");
+    }
+
+    @Test
+    public void validateMetaData() {
+        Assert.assertEquals(cell.getMetadata().getName(), "test-img12");
+        Assert.assertEquals(cell.getMetadata().getAnnotations().get(CELLERY_MESH + "/" + CELLERY_IMAGE_ORG),
```

Comment:
Instead of using separate variables and concatnate let's Use a singel variable `CELLERY_IMAGE_ORG`.